scaleway/resource_security_group_rule.go

```diff
@@ -94,11 +99,22 @@ func resourceScalewaySecurityGroupRuleCreate(d *schema.ResourceData, m interface
 	}
 
 	d.SetId(rule.ID)
-
 	if d.Id() == "" {
 		return fmt.Errorf("Failed to find created security group rule")
 	}
 
+	err = scaleway.PutSecurityGroupRule(api.UpdateSecurityGroupRule{
+		Position:     d.Get("position").(int),
+		Action:       d.Get("action").(string),
+		Direction:    d.Get("direction").(string),
+		IPRange:      d.Get("ip_range").(string),
+		Protocol:     d.Get("protocol").(string),
+		DestPortFrom: d.Get("port").(int),
+	}, d.Get("security_group").(string), rule.ID)
+	if err != nil {
+		return err
+	}
+
```

Comment:
Can't we just `return resourceScalewaySecurityGroupRuleUpdate(d, m)` instead of calling the update/Put method here? 😉

Reply:
yes we can. will update the PR.